### ISSUE TITLE
Problem installing on MacOS

### ISSUE THREAD

**Issue by k06a** [MrRefactoring/wasm-opt#1]
```
npm i wasm-opt -g
```
```
/Users/k06a/.npm-packages/bin/wasm-opt -> /Users/k06a/.npm-packages/lib/node_modules/wasm-opt/out/wasm-opt.js

> wasm-opt@1.1.0 postinstall /Users/k06a/.npm-packages/lib/node_modules/wasm-opt
> node ./out/index.js

✔ Release success found
✔ Binaries successfully downloaded
✔ Extracted
(node:39419) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, copyfile 'binaryen-1.39.1-x86_64-apple-darwin/wasm-opt' -> './out/wasm-opt'
    at Object.copyFileSync (fs.js:1728:3)
    at Object.<anonymous> (/Users/k06a/.npm-packages/lib/node_modules/wasm-opt/out/index.js:88:20)
    at step (/Users/k06a/.npm-packages/lib/node_modules/wasm-opt/out/index.js:31:23)
    at Object.next (/Users/k06a/.npm-packages/lib/node_modules/wasm-opt/out/index.js:12:53)
    at fulfilled (/Users/k06a/.npm-packages/lib/node_modules/wasm-opt/out/index.js:3:58)
(node:39419) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:39419) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
+ wasm-opt@1.1.0
updated 1 package in 23.35s
```

**Comment by MrRefactoring** [comment]
Same error for Linux

```bash
npm i -g wasm-opt
/home/osboxes/.nvm/versions/node/v13.5.0/bin/wasm-opt -> /home/osboxes/.nvm/versions/node/v13.5.0/lib/node_modules/wasm-opt/out/wasm-opt.js

> wasm-opt@1.1.0 postinstall /home/osboxes/.nvm/versions/node/v13.5.0/lib/node_modules/wasm-opt
> node ./out/index.js

✔️ Release success found
✔️ Binaries successfully downloaded
✔️ Extracted
(node:5171) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, copyfile 'binaryen-1.39.1-x86_64-linux/wasm-opt' -> './out/wasm-opt'
    at Object.copyFileSync (fs.js:1822:3)
    at Object.<anonymous> (/home/osboxes/.nvm/versions/node/v13.5.0/lib/node_modules/wasm-opt/out/index.js:88:20)
    at step (/home/osboxes/.nvm/versions/node/v13.5.0/lib/node_modules/wasm-opt/out/index.js:31:23)
    at Object.next (/home/osboxes/.nvm/versions/node/v13.5.0/lib/node_modules/wasm-opt/out/index.js:12:53)
    at fulfilled (/home/osboxes/.nvm/versions/node/v13.5.0/lib/node_modules/wasm-opt/out/index.js:3:58)
(node:5171) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag --unhandled-rejections=strict (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:5171) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
+ wasm-opt@1.1.0
added 31 packages from 29 contributors in 17.542s
```

**Comment by MrRefactoring** [comment]
Nice catch @k06a! Thank you. Fixed in `1.1.1` version
